docs/usage/RsyncConfiguration.md

```diff
@@ -0,0 +1,112 @@
+
+---
+
+### Rsync configuration using MigrationController CR
+
+_MigrationController_ CR exposes following variables to configure Rsync options in Direct Volume Migration (DVM):
+
+| Variable            	| Type   	| Default                                          	| Description                                                                              	|
+|---------------------	|--------	|--------------------------------------------------	|------------------------------------------------------------------------------------------	|
+| rsync_opt_bwlimit   	| int    	| Not set                                          	| When set to a positive integer, `--bwlimit=<int>` option will be added to Rsync command. 	|
+| rsync_opt_archive   	| bool   	| true                                             	| Sets `--archive` option in Rsync command.                                                	|
+| rsync_opt_partial   	| bool   	| true                                             	| Sets `--partial` option in Rsync command.                                                	|
+| rsync_opt_delete    	| bool   	| true                                             	| Sets `--delete` option in Rsync command.                                                 	|
+| rsync_opt_hardlinks 	| bool   	| true                                             	| Sets `--hard-links` option is Rsync command.                                             	|
+| rsync_opt_info      	| string 	| COPY2,DEL2,REMOVE2,SKIP2,FLIST2,PROGRESS2,STATS2 	| Enables detailed logging in Rsync Pod.                                                   	|
+| rsync_opt_extras    	| string 	| Empty                                            	| Reserved for any other arbitrary options.                                                	|
+
+
+Please note that the options set through the variables above are _global_ for all migrations. The configuration will take effect for all future migrations as soon as the operator successfully reconciles the _MigrationController_ CR. Any ongoing migration _may_ or _may not_ use the updated settings depending on which step it currently is in. Therefore, it is recommended that the settings be applied prior to running a migration. The users can always update the settings as needed.
+
+Please use `rsync_opt_extras` variable with extreme caution. Any options passed using this variable will be appended to Rsync command as-is. Make sure you add whitespaces when specifying more than one options. Any human error in specifying options _may_ lead to failed migrations. But thanks to Migration Operator, you can always _correct_ your mistakes by updating the _MigrationController_ CR as many times as you want for future migrations. 
+
+Please note that the default configuration used by DVM is tested in variouos environments and is deemed apt for most production use cases provided the clusters are healthy and performing well.
+
```

Comment:
@pranavgaikwad I think would like to add some explanation about the rsync_opt_info flag. The removal of `PROGRESS2` from the list of rsync_opt_info flag will likely lead to a situation there mig-controller will not be able to report any progress on the rsync transfer, I would suggest to use it only when the user knows what they are doing

Reply:
Fixed